Description: Fix jQuery compatibility.
 Bug#831332 icinga-cgi-bin: fix menu hiding (jquery 1.11 incompatibility)
 .
 icinga-cgi-bin in Debian Sid, Ubuntu Yakkety (and Ubuntu Xenial) is
 affected by an upstream bug where the left-hand menu in the Classic
 icinga web interface disappears immediately after the page finishes
 loading. Upstream has provided a fix and in my testing this resolves the
 issue (which is due to current code incompatibilty with jQuery versions
 in Debian and Ubuntu).
Author: Markus Frosch <lazyfrosch@debian.org>
Bug-Debian: https://bugs.debian.org/831332

--- a/html/js/menu.js
+++ b/html/js/menu.js
@@ -9,11 +9,14 @@ $(document).ready(function() {
 
 		elem.prepend( img )
 			.css( "cursor", "pointer" )
-			.toggle( function() {
-				img.attr('src', 'images/menu_more.gif');
-				list.slideToggle("slow");
-			}, function() {
-				img.attr('src', 'images/menu_less.gif');
+			.click( function() {
+				if($(this).hasClass("collapsed")) {
+					img.attr('src', 'images/menu_less.gif');
+					$(this).removeClass("collapsed");
+				} else {
+					img.attr('src', 'images/menu_more.gif');
+					$(this).addClass("collapsed");
+				}
 				list.slideToggle("slow");
 		});
 
